packages/ui/package.json

```diff
@@ -16,7 +17,11 @@
 		"react-router-dom": "^5.2.0"
 	},
 	"devDependencies": {
+		"@babel/preset-react": "^7.14.5",
 		"@vitejs/plugin-react-refresh": "^1.3.1",
+		"babel-jest": "^27.0.6",
+		"jest": "^27.0.6",
+		"react-test-renderer": "^17.0.2",
```

Comment:
You didn't use this (`react-test-renderer`) anywhere. 
I suggest you add an example so that others can follow it easily.